src/main/java/io/cdap/plugin/gcp/common/Schemas.java

```diff
@@ -59,8 +59,18 @@ public static void validateFieldsMatch(Schema trueSchema, Schema providedSchema)
                         field.getName(), providedFieldType.name(), trueFieldType.name()));
       }
       if (isTrueFieldNullable && !isProvidedFieldNullable) {
+        throw new IllegalArgumentException(String.format("Field '%s' should be nullable.", field.getName()));
```

Comment:
I don't think we need to throw an exception here. This is if the field in BigQuery is nullable but the data we're writing is not nullable, which should be fine.

Reply:
In that case, are assuming that users know their data and there are no null values present for that field?